package.json

```diff
@@ -3,8 +3,10 @@
   "version": "0.8.0",
   "author": "Mobify",
   "dependencies": {
+    "astro-sdk": "^0.9.0-dev",
     "bluebird": "~2.9.24",
-    "astro-sdk": "^0.9.0-dev"
+    "jquery": "^2.2.2",
+    "navitron": "https://github.com/mobify/navitron#develop"
```

Comment:
Github references can be shortened to simply `{org}/{repo}#branch`.   So this could be `mobify/navitron#develop`.  It's not a huge diff but tightens things up a bit and avoids hard-coding the transport protocol used.